translations/en.json

```diff
@@ -687,8 +687,8 @@
 	"mobile-syncing-sheet-details": "Status tends to use a lot of data when syncing chats. You can choose not to sync when on mobile network",
 	"mobile-syncing-sheet-title": "Sync using Mobile data",
 	"more": "more",
-	"multiaccount-exists-title": "Multiaccount already exists",
-	"multiaccount-exists-content": "This multiaccount already exists. You just need to unlock it",
+	"multiaccount-exists-title": "Keys already exists",
```

Comment:
Should it be singular - `exist`?

Reply:
Thanks @siphiuel! Your totally right about all of these. Sorry to ask, but can you check if you access the branch and update these yourself? I'm on vacation and AFK for at least the evening. Wouldn't want to delay merge.